src/views/Admin.js

```diff
@@ -30,13 +30,16 @@ import TableCell from '@material-ui/core/TableCell';
 import TableHead from '@material-ui/core/TableHead';
 import TableFooter from '@material-ui/core/TableFooter';
 import TablePagination from '@material-ui/core/TablePagination';
-
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEdit as faPencilAlt } from '@fortawesome/free-solid-svg-icons';
+import Signup from './Signup';
 ReactGa.initialize(process.env.REACT_APP_GAID, {
   testMode: process.env.NODE_ENV !== 'production',
 });
 
 const User = (props) => (
   <TableRow>
+    <TableCell>{null}</TableCell>
```

Comment:
why is there a null cell?

Reply:
its providing an additional cell for the plus button to add a new user. This is how they have it in prod for design purposes.

Reply:
You can remove the null and just leave it empy between the TableCell tags and get the same effect.

Reply:
Remove the `{null}`